Source/Events.Processing/Streams/StreamProcessor.cs

```diff
@@ -68,6 +68,20 @@ public class StreamProcessor : IDisposable
             _stopAllScopedStreamProcessorsTokenSource = CancellationTokenSource.CreateLinkedTokenSource(cancellationToken);
         }
 
+        /// <summary>
+        /// Get a specific underlying <see cref="AbstractScopedStreamProcessor"/> for a specific tenant.
+        /// </summary>
+        /// <param name="tenantId">Tenant to get for</param>
+        /// <returns><see cref="AbstractScopedStreamProcessor"/></returns>
+        public AbstractScopedStreamProcessor GetScopedStreamProcessorFor(TenantId tenantId)
+        {
+            if (!_streamProcessors.ContainsKey(tenantId))
+            {
+                throw new MissingScopedStreamProcessorTenant(tenantId);
+            }
+            return _streamProcessors[tenantId];
```

Comment:
I have some comments about event handlers and specific tenants later on. Would be nice if we don't have to have this exception here.